domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3157,6 +3168,17 @@ public void select(AsyncResult<SQLConnection> conn, String sql, Handler<AsyncRes
     }
   }
 
+  /**
```

Comment:
`Run an prepared/parameterized SQL statement that updates data and returns data, for example {@code SELECT nextval($1)} or {@code UPDATE ... RETURNING ...}.`

Reply:
I resolved this already in the latest commit but Github doesn't recognize the differences.